twingate/setup/mint-leaf.yml

```diff
@@ -1,7 +1,7 @@
 name: twingate/setup
-version: 1.0.0
+version: 1.0.1
 description: Install & setup Twingate
-source_code_url: https://github.com/rwx-research/mint-leaves/tree/main/twingate/install
+source_code_url: https://github.com/rwx-research/mint-leaves/tree/main/twingate/setup
```

Comment:
😨 thanks for catching this

Reply:
I just happened to click the link because I wanted to view the source 😊